src/i18n/man.json

```diff
@@ -0,0 +1,82 @@
+{
+  "locale": "man",
+  "lang_name": "简体中文",
+  "lang_name_in_eng": "Mandarin",
+  "language": "语言",
+  "welcome": "欢迎",
+  "welcome_blurb": "双城相助 （Twin Cities Mutual Aid）是关于明尼阿波利斯和圣保咯两座城人民市相助地点的最新目录。",
+  "lang_select": "选择语言",
+  "page_title": "双城相助地图",
+  "show_list_button": "显示地点单子",
+  "hide_list_button": "关闭地点单子",
+  "sort_by": "用类目分别",
+  "sort_by_urgent_need": "紧急请求",
+  "sort_by_name": "按照名称排序",
+  "sort_by_status": "地点现状",
+  "sort_by_neighborhood": "按照小区名称排序",
+  "sort_by_seeking_money": "需要钱",
+  "sort_by_seeking_volunteers_badge": "需要志愿者",
+  "filter_by_receiving": "能收捐献",
+  "filter_by_distributing": "能收捐献",
+  "filter_by_both": "能收到或分送捐献",
+  "filter_by_closed": "现在不能收到或分送捐献",
+  "filter_by_unknown": "没有数据",
+  "filter": "分别",
+  "receiving": "在接收",
+  "distributing": "在分送",
+  "both": "在接收和分送",
+  "closed": "已关闭",
+  "about": "关于",
+  "this_project": "这个项目",
+  "help_info": "支持中心/信息",
+  "project_team": "Twin Cities Mutual Aid Project",
+  "project_description": "双城相助地图是被志愿者管理为<a href=http://tcmap.org>Twin Cities Mutual Aid Project</a>之项目",
+  "project_learn": "发现更多信息在...",
+  "project_data": "在Google表格中查看这些资料",
+  "project_feedback": "评论和问题",
+  "project_contact": "发电子邮件",
+  "close": "关",
+  "name": "名称",
+  "neighborhood": "小区",
+  "address": "地址",
+  "currently_open_for_distributing": "现在在分送捐献",
+  "opening_for_distributing_dontations": "分送捐献时间: 公司开门",
+  "closing_for_distributing_donations": "分送捐献时间: 公司关门",
+  "accepting": "能受",
+  "not_accepting": "不能受",
+  "currently_open_for_receiving": "现在在收捐献",
+  "opening_for_receiving_dontations": "收捐献时间: 公司 开门",
+  "closing_for_receiving_donations": "收捐献时间: 公司关门门",
+  "opening_soon": "快要开门！",
+  "for_distribution": "分送",
+  "for_receiving": "收到",
+  "seeking_volunteers_badge": "寻找志愿者",
+  "seeking_money": "寻找捐款",
+  "seeking_money_link": "来捐款！",
+  "urgent_need": "急用",
+  "notes": "笔记",
+  "last_updated": "最近更新",
+  "aid_distribution_hours": "相助分送时间",
+  "aid_receiving_hours": "相助接受时间",
+  "time_future": "",
```

Comment:
This does not appear to have a translation, but I also don't see anywhere were it's used anyway...